src/include/parsing/utils/parsing_table.h

```diff
@@ -0,0 +1,147 @@
+#ifndef JUCC_PARSING_UTILS_PARSING_TABLE_H
+#define JUCC_PARSING_UTILS_PARSING_TABLE_H
```

Comment:
parser/parsing_table.h no need for additional folder
make necessary header guard change.